packages/lisk-api-client/test/api_resource.ts

```diff
@@ -125,6 +125,34 @@ describe('API resource module', () => {
 		});
 
 		describe('when response status is greater than 300', () => {
+			it('should reject with errno if status code is supplied', () => {
+				const statusCode = 300;
+				requestStub.rejects({
+					response: {
+						status: statusCode,
+						data: undefined,
+					},
+				});
+
+				return resource.request(defaultRequest, false).catch(err => {
+					return expect(err.message).to.equal('An unknown error has occurred.');
+				});
+			});
+
+			it('should reject with "An unknown error has occured." message if there is no data is supplied', () => {
```

Comment:
Is this test case any useful? Because in the following test, the request is rejected with a defined data property, and the expectation for `err.message` is the same.

Reply:
I think it's useful, because data exists but no message, and no data is treated in the different way in the code